bascule/basculehttp/log.go

```diff
@@ -0,0 +1,34 @@
+package basculehttp
+
+import (
+	"context"
+	"os"
+
+	"github.com/go-kit/kit/log"
+)
+
+var (
+	defaultLogger = log.NewJSONLogger(log.NewSyncWriter(os.Stdout))
+
+	callerKey    interface{} = "caller"
+	messageKey   interface{} = "msg"
+	errorKey     interface{} = "error"
+	timestampKey interface{} = "ts"
+)
+
+// logger we expect for the decorators
+type Logger interface {
```

Comment:
This should really be in the parent package, `bascule`.  Logging is a shared concept across the library.

I'd move this type and the default type and supporting functions up a package.  The option functions (e.g. `WithCLogger`) can stay here.